rdmo/projects/assets/js/interview/actions/contactActions.js

```diff
@@ -0,0 +1,84 @@
+import ContactApi from '../api/ContactApi'
+
+import { projectId } from '../utils/meta'
+
+import {
+  FETCH_CONTACT_INIT,
+  FETCH_CONTACT_SUCCESS,
+  FETCH_CONTACT_ERROR,
+  SEND_CONTACT_INIT,
+  SEND_CONTACT_SUCCESS,
+  SEND_CONTACT_ERROR,
+  CLOSE_CONTACT
+} from './actionTypes'
+
+import { addToPending, removeFromPending } from 'rdmo/core/assets/js/actions/pendingActions'
+
+export function fetchContact({ questionset, question, values }) {
+  const pendingId = 'fetchContact'
+
+  return (dispatch, getState) => {
+    const params = {
+      page: getState().interview.page.id,
```

Comment:
Is it not possible to get the texts here already from the React state and send those as params instead of only the ids??
It seems like redundant and slow to send only the ids and resolve those in the backend again to texts?!

Reply:
ok thanks, yes it's not critical in here and I would also keep the current implementation in place anyways.